pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -38,7 +38,7 @@ import (
 )
 
 func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance) error {
-	if err := l.hotPlugHostDevices(vmi); err != nil {
+	if err := l.HotplugHostDevices(vmi); err != nil {
```

Comment:
You have s spitted it from the merge finalize, so it should not appear here anymore.

Reply:
Good catch!

Reply:
Done